src/dualscreen.tsx

```diff
@@ -135,6 +127,7 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
   };
 
   private _removeActives() {
+    this._singleLayout = true;
```

Comment:
general question - do we support change media? all plugin flags and states should be reset when changing the media.